vespa-cloud/vespa-documentation-search/src/test/java/ai/vespa/cloud/docsearch/VespaDocSystemTest.java

```diff
@@ -53,26 +55,55 @@ public void testFeedSearchDelete() throws Exception {
         assertEquals(0, ids.size(), "visit all docs and remove");
     }
 
+    @Test
+    public void testUpdate() throws Exception {
+        updateTestDocs("/test-documents-updates.json");
+
+        HashSet<String> ids = getTestDocIDs();
+        ids.forEach(id -> {System.out.println(id);});
+        assertEquals(5, ids.size(), "test-documents-updates.json has 5 documents");
+
+        removeTestDocs(ids);
+        ids = getTestDocIDs();
+        assertEquals(0, ids.size(), "visit all docs and remove then");
+    }
+
     @Test
     public void testInlinks() throws Exception {
         feedTestDocs("/test-documents.json");
         HashSet<String> ids = getTestDocIDs();
         assertEquals(5, ids.size());
 
+        verifyInlinks();
+
+        removeTestDocs(ids);
+
+        updateTestDocs("/test-documents-updates.json");
+        ids = getTestDocIDs();
+        assertEquals(5, ids.size());
+
+        verifyInlinks();
+    }
```

Comment:
Should probably remove documents again here?

Reply:
yes, maybe. not important in system test env. Will add in next PR for symmetry

Reply:
Well, if this is run before some other test in the class, this may violate its assumptions.